docs/platform/howto/btar/manage-backup-to-another-region.md

```diff
@@ -0,0 +1,126 @@
+---
+title: Manage a cross-region backup
+sidebar_label: Manage cross-region backup
+pro: true
+---
+
+For a service that has the [backup to another region (BTAR) feature](/docs/platform/concepts/backup-to-another-region) enabled, you can check the service backup status, change the backup region, monitor the replication lag, fork and restore using [the cross-region backup](/docs/platform/concepts/backup-to-another-region), or migrate to another cloud or region.
+
+:::important
+Backup to another region (BTAR) is available on
+`[Pro Platform](/docs/platform/concepts/pro-platform)`.
+:::
+
+## Prerequisites
+
+You have at least one
+[Aiven service with BTAR enabled](/docs/platform/howto/btar/enable-backup-to-another-region).
+
+## Change a backup region
+
+1. Log in to the [Aiven Console](https://console.aiven.io/).
+1. From the **Services** view, select an Aiven service on which you'd like to enable BTAR.
+1. On your service's page, select **Backups** from the sidebar.
+1. On the **Backups** page, select the actions (**...**) menu > **Secondary backup
+   location**.
+1. In the **Edit secondary backup location** window, use the **Secondary location**
+   dropdown menu to select a region for your additional backup. Confirm your choice by
+   selecting **Save**.
+
+:::important
+Think carefully before you decide to change a backup region. Since each change of a backup
+region triggers a new base backup, you can change a backup region only infrequently.
```

Comment:
Let's not pretend readers don't already think carefully. What do you mean by infrequently? What's the consequence for me as a user, I can't switch back in the next 24h? 7days? Until the backup's over?

Consider something like this that's shorter and doesn't generate more questions?

```suggestion
Changing a backup region triggers a new base backup that can take up to X minutes. You cannot switch back until the backup is complete.
```